frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/FeatureOverviewMetaData.tsx

```diff
@@ -244,12 +244,17 @@ const FeatureOverviewMetaData = () => {
                     />
                 }
             />
-            <MarkCompletedDialogue
-                isOpen={showMarkCompletedDialogue}
-                setIsOpen={setShowMarkCompletedDialogue}
-                projectId={feature.project}
-                featureId={feature.name}
-                onComplete={refetchFeature}
+            <ConditionallyRender
+                condition={Boolean(feature.project)}
```

Comment:
same guard for the mark completed. We need project since hooks inside the dialogue assume it's not empty